portal-network.md

```diff
@@ -43,7 +43,13 @@ TODO
 
 ### State: Accounts and Contract Storage
 
-TODO
+The state network needs to make pieces of the state available to light clients before the next block is mined, so that they can be quickly retrieved by the users' wallets and dapps.
+
+Nodes should be able to choose how much state they want to store and share, and the network should provide a way to identify which nodes to query for a wanted portion of state. This is so that every node, no matter how small, can contribute to the health and robustness of the network.
+
+The network needs to have a way to push state from new blocks into the network, so that it can be made available for query and retrieval by interested nodes. Full "bridge" nodes acting as benevolent state providers would be responsible for bringing in this data from the main network, and the network should be able to remain healthy even with a small number of bridge nodes.
```

Comment:
```suggestion
The network will be dependent on receiving new and updated state from new blocks. Full "bridge" nodes acting as benevolent state providers would be responsible for bringing in this data from the main network. The network should be able to remain healthy even with a small number of bridge nodes.
```

Slight re-wording since "the network" doesn't actually pull this data in so I wanted to change phrasing a little to make it clear that this isn't something the network is doing itself.  Also broke up the sentences a little.

Reply:
Makes sense! I also changed `from new blocks` to `for new blocks` as it might be interpreted that blocks are somehow pushing the state. I guess `of new blocks` might also work, considering the state root is in the header? Up to you!